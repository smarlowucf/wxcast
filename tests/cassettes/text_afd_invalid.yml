interactions:
- request:
    body: null
    headers:
      Accept: ['*/*']
      Accept-Encoding: ['gzip, deflate']
      Connection: [keep-alive]
      User-Agent: ['wxcast app. https://github.com/smarlowucf/wxcast']
    method: GET
    uri: https://api.weather.gov/products/types/AFD/locations/FAKE
  response:
    body: {string: "{\n    \"type\": \"FeatureCollection\",\n    \"features\": []\n\
        }"}
    headers:
      Access-Control-Allow-Origin: ['*']
      Cache-Control: ['public, max-age=900, s-maxage=120']
      Connection: [keep-alive]
      Content-Length: ['55']
      Content-Type: [application/geo+json]
      Date: ['Fri, 01 Dec 2017 02:43:14 GMT']
      Expires: ['Fri, 01 Dec 2017 02:58:14 GMT']
      Server: [nginx]
      Strict-Transport-Security: [max-age=31536000; includeSubDomains; preload]
      Vary: [Accept]
      X-Correlation-ID: [79f4773e-1c4b-4dd4-b246-27b148296bdb]
      X-Request-ID: [79f4773e-1c4b-4dd4-b246-27b148296bdb]
      X-Server-ID: [vm-bldr-nids-apiapp67.ncep.noaa.gov]
    status: {code: 200, message: OK}
version: 1
