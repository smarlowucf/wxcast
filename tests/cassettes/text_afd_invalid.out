No wx data found attempting to retrieve afd issued by fake.
